modules/asg/variables.tf

```diff
@@ -288,6 +288,12 @@ variable "security_group_ids" {
   default     = []
 }
 
+variable "tags_dest" {
```

Comment:
```suggestion
variable "tag_specifications_targets" {
```
Can you please update the variable name to a more meaningful one?

Reply:
Thanks, updated it.